### PR TITLE
Fix initial facet distribution when using sortby

### PR DESCRIPTION
**Pull request by bidoubiwa**
Fixes: #1013 

Additionally I removed a file that was not used in the code-base and is basically an old version of `init-facets-distribution`

### CODE REVIEW

File: packages/instant-meilisearch/src/client/instant-meilisearch-client.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export function instantMeiliSearch(
           const adaptedSearchResponse = adaptSearchResponse<T>(
             searchResponse,
             searchContext,
-            initialFacetDistribution[searchRequest.indexName]
+            initialFacetDistribution[searchContext.indexUid]
```
Comment (brunoocasali) [inline packages/instant-meilisearch/src/client/instant-meilisearch-client.ts:101]: No test?! 👀

Comment (Strift) [reply packages/instant-meilisearch/src/client/instant-meilisearch-client.ts:101]: From my modest understanding of what's happening, it looks like the correct thing to do to retrieve the initial facet distribution.

**Review by Strift — COMMENTED** [review]
The solution looks good to me — but I'm unsure (and lacking knowledge to understand) what's the impact of the removed file.

File: packages/instant-meilisearch/src/client/instant-meilisearch-client.ts (hunk 1 above)
Comment (bidoubiwa) [reply packages/instant-meilisearch/src/client/instant-meilisearch-client.ts:101]: Done!

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**